src/Dfe.PlanTech.Web/Controllers/CheckAnswersController.cs

```diff
@@ -39,7 +38,8 @@ public class CheckAnswersController : BaseController<CheckAnswersController>
     [Route("change-answer")]
     public IActionResult ChangeAnswer(string questionRef, string answerRef, int submissionId)
     {
-        return RedirectToAction("GetQuestionById", "Questions", new { id = questionRef, answerRef = answerRef, submissionId = submissionId });
+        TempData[TempDataConstants.Questions] = Newtonsoft.Json.JsonConvert.SerializeObject(new TempDataQuestions() { QuestionRef = questionRef, AnswerRef = answerRef, SubmissionId = submissionId });
```

Comment:
Should this not use the `DeserialiseParameter` method on the `BaseController`?

Reply:
This part is the serialisation of the parameters into the JSON string before calling the redirect of the other controller to call, where the called controller then deserialises them on the other end into the appropriate object. Saying that, I should make a `SerialiseParameter(..)` method in `BaseController` to make this more obvious

Reply:
Added `SerialiseParameter(..)` in [7965491](https://github.com/DFE-Digital/plan-technology-for-your-school/pull/216/commits/79654910da2973c10a11a4197cb26b14d4c77b87)